src/utils/data/chains.ts

```diff
@@ -18,7 +19,12 @@ export function getSupportedChainIds(): Array<string> {
 export function getDefaultChainId(): string {
     return getSupportedChainIds()[0];
 }
-
+export function getDefaultPairForChain(chainId: string): [TokenIF, TokenIF] {
```

Comment:
Surfacing the new home of this function - placing in defaultTokens caused a circular import issue.